src/main/java/lotto/model/LottoResult.java

```diff
@@ -0,0 +1,42 @@
+package lotto.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoResult {
+
+    private List<Winner> winnerResult = new ArrayList<>();
+    private List<List<Integer>> lottoNumbers;
+    private String[] inputLastNumber;
```

Comment:
inputLastNumber가 정확히 어떤 의미인지 알기가 쉽지 않습니다 ㅠ
코드를 보면, 로또 당첨 번호를 의미하는 것 같은데, 
```suggestion
    private String[] winningNumber;
```
등으로 이름을 바꾸는게 어떨까요? 그리고 이 또한 마찬가지로 List<Integer> 형태를 가져도 될 것 같습니다:)